src/workloads/execution/TimeSeriesLastpoint.yml

```diff
@@ -46,75 +111,309 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
 
-- Name: CreateIndex
+# Lastpoint query with a sort and group with meta subfield descending and time ascending, but no index.
+- Name: RunLastpointQueryWithMetaSubfieldAscendingTimeDescendingNoIndex
```

Comment:
Question: was there a reason for having the index/no index as separate actors? Would it be possible to instead do something like below instead? If so it would simplify the workload and help with readability.

```
- Name: RunLastpointQueryWithMetaSubfieldAscendingTimeDescending
    # No index operations (Quiesce, LastpointQueryMetaAscTimeDesc)
    # With index (CreateIndex, Quiesce, LastpointQueryMetaAscTimeDesc, DropIndex)
```

Reply:
I don't think there is any clean way to do this because of phases, but I'll ask in perf-tooling-users to see if anyone knows of a better way to do this.

There are ActorTemplates which allow us to parametrize entire actors; however, because each actor has an array of phases such that the actual workload begins at a different index, there is no way to parametrize on the actor level without having all the workloads occur simultaneously (hence the giant array of Nops).

Similarly, there are phases; however, afaik, we can't parametrize arrays of phases, only single phases. This makes the repetition necessary (and unfortunate). Writing a dedicated actor to handle this seemed like overkill for this ticket.

Reply:
Ah gotcha, thanks for expanding on that - in that case the workload structure make sense to me.